### PR TITLE
Console output cleanup

### PR DESCRIPTION
**Pull request by proxict**
- Changed some output printing to only print in debug (was a debug output anyway)
 - Improved game-chat readability (colors depicting teams)

### CODE REVIEW

File: src/sv_snapshot.c [hunk 1]
```diff
@@ -161,16 +161,16 @@ __cdecl void SV_WriteSnapshotToClient(client_t* client, msg_t* msg){
         client->demoDeltaFrameCount--;
 
         if(oldframe->first_entity <  svsHeader.nextSnapshotEntities - svsHeader.numSnapshotEntities) {
-            Com_PrintWarning(CON_CHANNEL_SERVER,"%s: Delta request from out of date entities - delta against entity %i, oldest is %i, current is %i.  Their old snapshot had %i entities in it\n",
-                            client->name, oldframe->first_entity, svs.nextSnapshotEntities - svs.numSnapshotEntities, svs.nextSnapshotEntities, oldframe->num_entities );
+            // Com_PrintWarning(CON_CHANNEL_SERVER,"%s: Delta request from out of date entities - delta against entity %i, oldest is %i, current is %i.  Their old snapshot had %i entities in it\n",
```
Comment (Iswenzz) [inline src/sv_snapshot.c:164]: These warnings are useful for me, should make them a debug print

File: src/g_sv_cmds.c [hunk 2]
```diff
@@ -618,7 +619,16 @@ __cdecl void G_Say(gentity_t *ent, gentity_t *target, int mode, const char *chat
...
+    const int isPlaying = ent->client->sess.sessionState == SESS_STATE_PLAYING;
+    const int isAttack = ent->client->sess.cs.team == TEAM_BLUE;
+    const int isDefence = ent->client->sess.cs.team == TEAM_RED;
+    const int c1 = isAttack ? 1 : 0;
+    const int c2 = isAttack ? 34 : (isDefence ? 33 : 0);
+    if(com_ansiColor && com_ansiColor->integer) {
+        Com_Printf(CON_CHANNEL_SERVER,"\033[%d;%dm%s%s%s\033[0m: %s\n", c1, c2, isPlaying ? "" : "(Dead)", mode == SAY_ALL ? "" : (isAttack ? "(Attack)" : "(Defence)"), name, text);
+    } else {
+        Com_Printf(CON_CHANNEL_SERVER,"%s%s%s: %s\n", isPlaying ? "" : "(Dead)", mode == SAY_ALL ? "" : (isAttack ? "(Attack)" : "(Defence)"), name, text);
+    }
```
Comment (Iswenzz) [inline src/g_sv_cmds.c:631]: Should be optional

Comment (proxict) [reply src/g_sv_cmds.c]: Made optional in 75af5e23fcdc56ebb621fc5b4135e82ec342c5b7

File: src/sv_snapshot.c (hunk 1 above)
Comment (proxict) [reply src/sv_snapshot.c]: Addressed in b3c80fd9bc232f84b29140081a040e8bf242d526

**Comment by proxict** [comment]
Thanks for the review, @Iswenzz!

**Comment by proxict** [comment]
@IceNinjaman Can we merge this?

**Pull request merged**